### PR TITLE
impr(quotes): add 8 new English quotes (LifeAdventurer)

### PR DESCRIPTION
**Pull request by LifeAdventurer**

### CODE REVIEW

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -38718,6 +38718,54 @@
...
+      "text": "Loss is nothing else but change, and change is nature's delight.",
+      "source": "Marcus Aurelius",
+      "length": 64,
+      "id": 7026
+    },
+    {
+      "text": "If there is magic on this planet, it is contained in water.",
+      "source": "Loren Eiseley",
+      "length": 59,
+      "id": 7027
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38756]: Remove, seems a bit weird without some context.

File: frontend/static/quotes/english.json [hunk 2]
```diff
@@ -38718,6 +38718,54 @@
...
+    },
+    {
+      "text": "If the world's a veil of tears, smile till rainbows span it.",
+      "source": "Lucy Larcom",
+      "length": 60,
+      "id": 7028
+    },
+    {
+      "text": "Life is in a land full of thorns and weeds, there always a space in which the good seed can grow.",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38765]: Seems like there is a "but" missing? "but there's" ?

File: frontend/static/quotes/english.json [hunk 3]
```diff
@@ -38718,6 +38718,54 @@
...
+      "id": 7024
+    },
+    {
+      "text": "Snow isn't just pretty. It also cleanses our world, our senses, and a kind of weary familiarity.",
+      "source": "John Burnside",
+      "length": 96,
+      "id": 7025
+    },
+    {
+      "text": "Loss is nothing else but change, and change is nature's delight.",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38747]: Remove

**Review by Miodec — CHANGES_REQUESTED** [review]

File: frontend/static/quotes/english.json (hunk 2 above)
Comment (LifeAdventurer) [reply frontend/static/quotes/english.json:38765]: I checked it, and there is a "is" missing, but no "but" missing.

File: frontend/static/quotes/english.json (hunk 3 above)
Comment (LifeAdventurer) [reply frontend/static/quotes/english.json:38747]: May I know why? I can be more careful next time and know how to choose.

File: frontend/static/quotes/english.json (hunk 1 above)
Comment (LifeAdventurer) [reply frontend/static/quotes/english.json:38756]: I have seen this quote from many websites.
https://www.goodreads.com/quotes/427330-if-there-is-magic-on-this-planet-it-is-contained
https://themeaningofwater.com/2014/12/01/water-is-magic/
https://www.azquotes.com/quote/87464

File: frontend/static/quotes/english.json (hunk 2 above)
Comment (NipulM) [reply frontend/static/quotes/english.json:38765]: The only similar quote I found, with a length of 125 characters, is: "Although the life of a person is in a land full of thorns and weeds, there is always a space in which the good seed can grow."

Comment (LifeAdventurer) [reply frontend/static/quotes/english.json:38765]: yeah, it's the same one. I saw "Life is in a land full of thorns and weeds, there is always a space in which the good seed can grow." on an app.

maybe I'll change it to this one. thanks~

File: frontend/static/quotes/english.json (hunk 3 above)
Comment (Miodec) [reply frontend/static/quotes/english.json:38747]: Just don't really like its content. Hard to explain.

Comment (LifeAdventurer) [reply frontend/static/quotes/english.json:38747]: okay, I removed it

**Review by NipulM — APPROVED** [review]
Reviewed the quotes, and they check out. Lengths match as specified. All good!

**Comment by LifeAdventurer** [comment]
Thanks for reviewing my pull request.

**Review by Miodec — APPROVED** [review]

**Comment by Miodec** [comment]
Please resolve conflict

**Comment by Miodec** [comment]
> Reviewed the quotes, and they check out. Lengths match as specified. All good!

Just so you know you don't need to check lengths yourself, they are checked automatically.

**Comment by LifeAdventurer** [comment]
> Please resolve conflict

Resolved

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/7568770486

**Comment by LifeAdventurer** [comment]
> Please resolve conflict

resolved

**Pull request merged**